recipes/pytest-workflow/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pytest-workflow" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c855171d1a68801e6cb61befb91faf1c33f4713c36a3e18e40081b0f85e41cd5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - jsonschema
+    - pip
+    - pytest >=4
+    - python >=3.5
+    - pyyaml
+    - setuptools
+  run:
+    - jsonschema
+    - pytest >=4
+    - python >=3.5
+    - pyyaml
+
+test:
+  imports:
+    - pytest_workflow
+
+about:
+  home: https://github.com/LUMC/pytest-workflow
+  license: GNU Affero General Public v3 or later (AGPLv3+)
+  license_family: AGPL
+  license_file: 
+  summary: A pytest plugin for configuring workflow/pipeline tests using YAML files
+  doc_url: https://pytest-workflow.readthedocs.io
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - rhpvorderman
```

Comment:
You can get rid of the `extra:` section. Also, can you remove the empty `dev_url:` and `license_file:` lines? Aside from that it looks good!